engine/src/main/java/org/hibernate/search/engine/search/highlight/dsl/QueryHighlightConfigurationOptionStep.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.highlight.dsl;
+
+/**
+ * TODO
+ */
+public interface QueryHighlightConfigurationOptionStep<T extends QueryHighlightConfigurationOptionStep<?>>
+		extends CommonHighlightOptionStep<T> {
+
+	T styledTagsSchema();
```

Comment:
Two things about options in the DSL in general (not just this one):

1. I think the names should be structured differently to allow easy discovery through IDE auto-completion, especially when there are multiple options and the IDE sorts them alphabetically. I understand that English requires adjectives to come first, but in this case that's just annoying. So `.tagsSchemaStyled()` would be much easier to discover, assuming there were other options such as `.tagsSchemaNone()`.
2. If we're going to allow per-field overrides, we need way to disable options as well as enable them. So either we introduce another method `noTagsSchema()`/`.tagsSchemaNone()`, or we just introduce an enum and turn this method into `.tagsSchema(HighlightingTagsSchema.STYLED)`. My preference goes to the latter in this case, but YMMV.